src/workers/parser/getSymbols.js

```diff
@@ -33,7 +34,7 @@ export type SymbolDeclaration = {
 };
 
 export type ClassDeclaration = SymbolDeclaration & {
-  parent: string
+  parent: Object | null
```

Comment:
we should improve the flow type here:

```
name: string,
location: Location
```

Reply:
Thanks for the feedback Jason, I would work on it this week. :)